mmv1/third_party/terraform/services/eventarc/resource_eventarc_channel_test.go

```diff
@@ -71,6 +77,10 @@ resource "google_eventarc_channel" "primary" {
   name                 = "tf-test-name%{random_suffix}"
   crypto_key_name      = "%{key2}"
   third_party_provider = "projects/%{project_name}/locations/%{region}/providers/datadog"
+
+  labels = {
```

Comment:
Can we remove it from the update config to test the update functionality of this field?